C/include/c4Query.h

```diff
@@ -101,14 +102,11 @@ extern "C" {
         The fields of this struct represent the current matched index row, and are valid until the
         next call to c4queryenum_next or c4queryenum_release. */
     struct C4QueryEnumerator {
-        /** The columns of this result, in the same order as in the query's `WHAT` clause. */
+        /** The columns of this result, in the same order as in the query's `WHAT` clause.
+            A column whose value is MISSING will have a Fleece 'undefined' value; this can be detected
+            by `FLValue_GetType(col) == kFLUndefined`. */
         FLArrayIterator columns;
 
-        /** A bitmap where a 1 bit represents a column whose value is MISSING.
-            This is how you tell a missing property value from a value that's JSON 'null',
-            since the value in the `columns` array will be a Fleece `null` either way. */
-        uint64_t missingColumns;
```

Comment:
Just pointing out this additional API change: instead of the `missingColumns` kludge, columns whose value is `MISSING` now appear as Fleece values of type `kFLUndefined`. (See the updated doc-comment for `columns`.)